### DISCUSSION TITLE
Ability to ad modifier button to an external device

### DISCUSSION THREAD

**Discussion by jordansoper** [Valkirie/HandheldCompanion#341] [Ideas]
Hello

Love the software and use it every time i use my onexplayer mini.

The main thing i use with it is the motion controls, i usually have it assigned to the left trigger so when i aim down sights i can use motion controls to fine-tune my aim.

What i am looking at doing is creating a rear button mod for the onexplayer to add 2 rear buttons to the device.

What i am looking for is the ability to use one of those rear buttons to activate the motion controls.

I would start my testing with a one-button keyboard, so haveing the ability to assign that button to the modifier button would be great.

Thanks

**Comment by CasperH2O** [comment]
@jordansoper nice idea! If the rear buttons will function as keyboard inputs, you could:
- Configure this keyboard key in Steam as trigger, use HC with a virtual DS4
- Wait for the remapper release, that should allow you to mix and match
- Adjust the device specific special keys of the OneXPlayer model you have and add your own in there. Although, sadly right now there seems to be a bug where special keys can't be used for UMC.

Please share your rear button result when you have something!

**Comment by CasperH2O** [comment]
@jordansoper HC has several options for enabling/disabling gyro:
- hold to either disable or enable motion
- toggle for motion on off, press the button only once
- aiming down sights multiplier with additional button options

Those should suffice for the use case you describe.
